messaging/pom.xml

```diff
@@ -0,0 +1,81 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Messaging module should be linked to the parent pom.xml as well. Otherwise CI does not build it.

Reply:
updated pom as per feedback